packages/v4/src/pages/index.js

```diff
@@ -162,11 +162,14 @@ const IndexPage = ({ location }) => {
     </PageSection>
     <PageSection variant={PageSectionVariants.darker} className="pf4-c-background-image-principles ws-homepage-main-section">
       <Grid gutter="md" className="pf-u-py-3xl">
-        <GridItem span={12}>
+        <GridItem sm={12} md={6} mdOffset={3} className="pf-u-text-align-center">
           <TextContent>
             <Text component={TextVariants.h1} className="pf-u-mb-4xl pf-m-white pf-u-text-align-center" id="patternfly-principles">
               {patternFlyPrinciples}
             </Text>
+            <Text component={TextVariants.p} className="ws-mdx-p">
+              Every Flyer is unique, with different ideas and opinions. But there are some things we all agree on. These are our Flyer fundamentals.
```

Comment:
Hi Abi, again, I'll leave this up to you, but I would expect this to read with no comma: "Every Flyer is unique with different ideas and opinions."

Reply:
This is a good thing to point out! I put the comma there because the "with different ideas and opinions" is sort of like an addition, a parenthetical. The goal is to make a statement "Every Flyer is unique," then add "with different ideas and opinions" as an addition (pausing where the comma is).